doc/release-notes-14707.md

```diff
@@ -0,0 +1,17 @@
+Wallet `receivedby` RPCs now include coinbase transactions
+-------------
+
+Previously, the following wallet RPCs incorrectly filtered out coinbase transactions:
```

Comment:
```suggestion
Previously, the following wallet RPCs excluded coinbase transactions:
```

Reply:
I agree that "filter" isn't the best term, because it's _slightly_ ambiguous. Maybe it's just me, but while it _probably_ means "exclude", it could possibly mean "include".